### PR TITLE
Create Logging JSON Codestart

### PR DESCRIPTION
**Pull request by jantosi**

### CODE REVIEW

**Review by ia3andy — CHANGES_REQUESTED** [review]
You can't use `application.properties`, you need to use `application.yml`, I described why in the doc: https://github.com/quarkusio/quarkus/blob/master/independent-projects/tools/codestarts/README.adoc#tips-for-writing-extension-example-code

Comments are going to be ignored, so I suggest you to document those configuration keys in the readme instead :)

File: devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/examples/logging-json/codestart.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: logging-json-example
+ref: logging-json
+type: code
+tags: example
+language:
+  base:
+    dependencies:
+      - io.quarkus:quarkus-logging-json
```
Comment (ia3andy) [inline devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/examples/logging-json/codestart.yml:8]:
```suggestion
      - io.quarkus:quarkus-resteasy
      - io.quarkus:quarkus-logging-json
```

Comment (ia3andy) [reply devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/examples/logging-json/codestart.yml:8]: This should fix the test which are not passing.

**Review by ia3andy — CHANGES_REQUESTED** [review]

**Review by gsmet — DISMISSED** [review]
The history needs to be cleaned up.

File: integration-tests/devtools/src/test/java/io/quarkus/devtools/codestarts/QuarkusCodestartRunSingleIT.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+import com.google.common.collect.ImmutableList;
+
+import io.quarkus.devtools.PlatformAwareTestBase;
+import io.quarkus.devtools.ProjectTestUtil;
+import io.quarkus.devtools.codestarts.QuarkusCodestartData.DataKey;
+import io.quarkus.devtools.messagewriter.MessageWriter;
+import io.quarkus.devtools.project.BuildTool;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class QuarkusCodestartRunSingleIT extends PlatformAwareTestBase {
```
Comment (ia3andy) [inline integration-tests/devtools/src/test/java/io/quarkus/devtools/codestarts/QuarkusCodestartRunSingleIT.java:25]: You can remove that class, your codestart is already automatically tested by the  `QuarkusCodestartRunIT`. I am currently creating a tool to easily generate a codestart in development which will remove the need of using the tests for this..

**Review by ia3andy — CHANGES_REQUESTED** [review]

**Review by ia3andy — APPROVED** [review]
🚀

**Pull request merged**